rpm/sailfish-components-webview.spec

```diff
@@ -1,5 +1,5 @@
 Name: sailfish-components-webview-qt5
-License: MPLv2
+License: MPLv2.0
```

Comment:
Url missing.

Please amend under the license:

```Url:        https://github.com/sailfishos/sailfish-components-webview```

So that column of the "h" of the https start from the same column as MPLv2.0